x-pack/filebeat/module/suricata/eve/test/eve-alerts.log-expected.json

```diff
@@ -16,6 +16,7 @@
         "http.request.method": "GET", 
         "http.response.status_code": "200", 
         "input.type": "log", 
+        "log.file.path": "/Users/ruflin/Dev/gopath/src/github.com/elastic/beats/x-pack/filebeat/module/suricata/eve/test/eve-alerts.log", 
         "log.offset": 0, 
```

Comment:
As this field is ignored by test_modules.py, can we get rid of this entries?

Reply:
Good question. Will need to investigate. Will also tackle this in one of the other PR's that is conflicting with this one as soon as it's merged if that is ok with you?

Reply:
You got lucky, have to rebase now anyways. Will fix these issues.